apps/guardian-ui/src/languages/es.json

```diff
@@ -128,7 +128,7 @@
       },
       "run-dkg": {
         "subtitle": "Todos los Guardianes han validado los detalles de la configuración de la Federación. Realizando algunos cálculos...",
-        "title": "¡Boom! Compartiendo información entre los Guardianes"
+        "title": "¡Éxito!"
```

Comment:
i'd remove these entries and let the bot auto translate for us so we have a consistent source of translations

Reply:
for es and ko

Reply:
We don't yet have the auto-translation code checked in so we don't yet have a canonical auto-translator. I think until then doing it ad-hoc is fine, but we should really get that formalized.

Reply:
#312